src/client/java/teammates/client/scripts/scalabilitytests/InstructorFeedbackResultsPageScaleTestDataGenerator.java

```diff
@@ -0,0 +1,173 @@
+        private String email = "CFResultsUiT.";
+        private String name;
+        private String section = "Section A";
+        private String team = "Team 1";
+
+        Student(String name) {
+            this.name = name;
+            this.googleId += name;
+            this.email += name + ".tmt";
+        }
+
+        public String getGoogleId() {
+            return googleId;
+        }
+    }
+
+    class Response {
+        private String giver;
+        private String recipient;
+        private String feedbackQuestionId;
+        private String courseId = "CFResultsUiT.CS2104";
+        private String feedbackQuestionType = "TEXT";
+        private String feedbackSessionName = "Open Session";
+        private String giverSection = "Section A";
+        private String recipientSection = "Section A";
+        private Metadata responseMetaData = new Metadata("Response.");
+
+        Response(String giver, String recipient, String feedbackQuestionId) {
+            this.giver = giver;
+            this.recipient = recipient;
+            this.feedbackQuestionId = feedbackQuestionId;
+        }
+    }
+
+    InstructorFeedbackResultsPageScaleTestDataGenerator(int numQuestions, int numStudents) {
+        courses.put("CFResultsUiT.CS2104", new Course());
+        accounts.put("CFResultsUiT.instr", new Account());
+        feedbackSessions.put("Open Session", new Session());
+
+        RandomNameGenerator nameGenerator = new RandomNameGenerator();
+        for (int i = 0; i < numStudents; i++) {
+            String name = nameGenerator.next();
+            students.put(name.replace("_", " "), new Student(name.replace("_", ".")));
+        }
+
+        int count = 0;
+        for (int i = 0; i < numQuestions; i++) {
+            feedbackQuestions.put("question" + i, new Question(i));
+            for (Student giver : students.values()) {
+                for (Student recipient : students.values()) {
+                    feedbackResponses.put(
+                            "response" + count,
+                            new Response(giver.getGoogleId(), recipient.getGoogleId(), String.valueOf(i)));
+                    count++;
+                }
+            }
+        }
+    }
+
+    public static void main(String[] args) throws IOException {
+        int[] studentNums = {10, 20};
+        int[] questionNums = {1, 5, 10};
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        for (int studentNum : studentNums) {
+            for (int questionNum : questionNums) {
+                Writer writer = new FileWriter(
+                        "src/client/java/teammates/client/scripts/scalabilitytests/"
+                        + "InstructorFeedbackResultsPageScaleTest-" + studentNum
+                        + "Students" + questionNum + "Questions.json");
+                gson.toJson(new InstructorFeedbackResultsPageScaleTestDataGenerator(questionNum, studentNum), writer);
+            }
+        }
+    }
+}
+
+
```

Comment:
I don't usually pick on this, but there are unnecessary empty lines at the end of this file :p